pkg/cmd/server/origin/master_config.go

```diff
@@ -575,19 +575,19 @@ func newAdmissionChain(pluginNames []string, admissionConfigFilename string, plu
 			plugin = saAdmitter
 
 		default:
-			configFile, err := pluginconfig.GetPluginConfigFile(pluginConfig, pluginName, admissionConfigFilename)
+			configFile, err := pluginconfig.GetAdmissionConfigurationFile(pluginConfig, pluginName, admissionConfigFilename)
```

Comment:
the upstream struct has the ability to inline a config blob as well... we should only need a single temp file that adapts our entire admission config stanza to the upstream AdmissionConfigurations and writes it out. I think a follow-up upstream PR to let us pass an AdmissionConfigurations struct to InitPlugins would let us remove any temp files needed.